From c996c9f52ced83a6f7b837fdc742e9b23e53a8a2 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Fri, 27 Mar 2020 00:46:45 +0500
Subject: [PATCH 16/16] conf: USB-Audio: Fix S/PDIF output of ASUS Xonar AE

ASUS Xonar AE is a PCI-Express card containing USB controller:

	USB controller [0c03]: ASMedia Technology Inc. ASM1042A USB 3.0 Host Controller [1b21:1142]

and the actual USB sound card:

	ID 0b05:180f ASUSTek Computer, Inc. XONAR SOUND CARD

As other Xonar USB sound cards, it uses second device for digital
output.

Signed-off-by: Ivan Mironov <mironov.ivan@gmail.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 src/conf/cards/USB-Audio.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/conf/cards/USB-Audio.conf b/src/conf/cards/USB-Audio.conf
index 89811086db1d..3f4c421b7e2c 100644
--- a/src/conf/cards/USB-Audio.conf
+++ b/src/conf/cards/USB-Audio.conf
@@ -40,6 +40,7 @@ USB-Audio.pcm.iec958_device {
 	"USB Sound Blaster HD" 1
 	"Xonar U7" 1
 	"ASUS XONAR U5" 1
+	"XONAR SOUND CARD" 1
 	
 	# The below don't have digital in/out, so prevent them from being opened.
 	"Andrea PureAudio USB-SA Headset" 999
-- 
2.16.4

